.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
I suspect most people will not delete any content and would instead start typing at the next line. Should such prompt/description comments like the one on this line be commented out? This way the bug reporter would not need to delete it and it would also not be left in in case they did not do so.